### DISCUSSION TITLE
Search icon instead of search filed on BSv6

### DISCUSSION THREAD

**Discussion by balrogmedia** [bootscore/bootscore#784] [Q&A]
Hello,
by default my search on the navbar has the search field:
![Screenshot 2024-05-27 at 11 40 40](https://github.com/bootscore/bootscore/assets/98542232/3a7b3afb-47c3-4248-936a-71b094ff62cf)

but I like more the the option of the icon only, with the search filed opening under the navbar like here
![Screenshot 2024-05-27 at 11 41 31](https://github.com/bootscore/bootscore/assets/98542232/b1ac6b16-c776-48bb-a3b4-51300cb16fbd)

would you be so kind to let me know how to achieve this?

Thank you for your time

**Comment by crftwrk** [comment]
Answered here https://github.com/orgs/bootscore/discussions/290#discussioncomment-9568590 as well.

Go to bootscores root folder to `template-parts/header/` and override `actions.php` and `collapse-search.php` by copy them to your child using the same folder structre `template-parts/header/[file].php`.

Now replace entire content:

#### actions.php

```php
<?php

/**
 * Template part for displaying the header-actions
 *
 * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
 *
 * @package Bootscore
 * @version 6.0.0
 */


// Exit if accessed directly
defined('ABSPATH') || exit;

?>

<!-- Search toggler -->
<?php if (is_active_sidebar('top-nav-search')) : ?>
  <button class="<?= apply_filters('bootscore/class/header/button', 'btn btn-outline-secondary', 'search-toggler'); ?> ms-1 ms-md-2 search-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-search" aria-expanded="false" aria-controls="collapse-search">
    <i class="fa-solid fa-magnifying-glass"></i><span class="visually-hidden-focusable">Search</span>
  </button>
<?php endif; ?>
```

#### collapse-search.php


```php
<?php

/**
 * Template part for displaying the top-nav searchform collapse widget
 *
 * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
 *
 * @package Bootscore
 * @version 6.0.0
 */


// Exit if accessed directly
defined('ABSPATH') || exit;

?>


<!-- Collapse Search -->
<?php if (is_active_sidebar('top-nav-search')) : ?>
  <div class="collapse <?= apply_filters('bootscore/class/header/collapse', 'bg-body-tertiary position-absolute start-0 end-0'); ?>" id="collapse-search">
    <div class="<?= apply_filters('bootscore/class/container', 'container', 'collapse-search'); ?> pb-2">
      <?php dynamic_sidebar('top-nav-search'); ?>
    </div>
  </div>
<?php endif; ?>

```

Solved?

**Reply by balrogmedia** [reply]
works great, thank you!

**Reply by crftwrk** [reply]
You're welcome 👍. Please mark answer as solved.

**Comment by crftwrk** [comment]
I reopen this, because other users may have the same question.
